desktop/api/src/main/java/org/datacleaner/util/WidgetUtils.java

```diff
@@ -655,4 +655,21 @@ public static void setDefaultButtonStyle(AbstractButton b) {
             b.setUI(DefaultButtonUI.get());
         }
     }
-}
\ No newline at end of file
+    
+    public static void disableComponent(JComponent component) {
```

Comment:
No, normally you have a UI class which acts similar to a CSS document/stylesheet. We have for instance DCComboBoxUI, PrimaryButtonUI, DarkButtonUI etc.

This is necesary because not every button or every text field should have the same style, so you don't want it to be applied to _every_ such button or text field.